lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -121,3 +121,19 @@ OperatingSystem _detectOperatingSystem() {
     return OperatingSystem.unknown;
   }
 }
+
+/// List of Operating Systems we know to be working on laptops/desktops.
+///
+/// These devices tend to behave differently on many core issues such as events,
+/// screen readers, input devices.
+List<OperatingSystem> _desktopOperatingSystems = [
+  OperatingSystem.macOs,
```

Comment:
nit: the system should be named `macOS`, if you're in control of these enum names

Reply:
I think @yjbanov suggested the opposite in a previous PR. I can change in the next PR if we agree on macOS though